telemetry/specs/INT.mdk

```diff
@@ -1309,7 +1326,7 @@ INT-MX Header:
 
 * The INT-MX header is 12 bytes long. Each metadata requested in the
   INT-MX Header instruction is either 4 bytes or 8 bytes in length. Each INT
-  capable device in the forwarding path will send the requested metadata to the
+  node in the forwarding path will send the requested metadata to the
```

Comment:
Below we have a link to TR spec 2.0 PDF. It makes sense for INT 2.1 spec.

In the master version of .mdk, we'd want to link to TR.mdk, not a particular PDF. There must be a way to define a variable in mdk, so that we can control INT version and matching TR version in one place.